packages/flutter/lib/src/widgets/editable_text.dart

```
@@ -2234,7 +2234,10 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     _lastBottomViewInset = WidgetsBinding.instance!.window.viewInsets.bottom;
   }
 
-  void _formatAndSetValue(TextEditingValue value, SelectionChangedCause? cause, {bool userInteraction = false}) {
+  // This method is often called by platform message handlers that catch and
+  // send unrecognized exceptions to the engine/platform. Make sure the
+  // exceptions user callbacks throw are handled within this method.
```

Nit:

```diff
- exceptions user callbacks throw
+ exceptions that user callbacks throw
```

That just makes it slightly easier to read IMO.

---

This swallowing of errors has bit me before and is hard to notice.